spec/rubocop/cop/rspec/excessive_docstring_spacing_spec.rb

```diff
@@ -120,17 +120,17 @@
 
     it 'skips \-separated multiline strings whose trailing whitespace ' \
        'makes sense' do
-      expect_no_offenses(<<-RUBY)
-        describe '#mymethod ' \\
+      expect_no_offenses(<<-'RUBY')
+        describe '#mymethod ' \
             '(is cool)' do
         end
```

Comment:
```
<<-A
a \
b
A
=> "a b\n"
```
The ` \` part is lost, and that undermines the very existence of this example which is purposed to test "\-separated multiline strings".

Reply:
The results seem to be the same here as well.

```ruby
irb(main):001:0" <<-A
irb(main):002:0" a \\
irb(main):003:0" b
irb(main):004:0> A
=> "a \\\nb\n"
irb(main):005:0' <<-'A'
irb(main):006:0' a \
irb(main):007:0' b
irb(main):008:0> A
=> "a \\\nb\n"
```